jsconfig-example.json

```diff
@@ -8,8 +8,12 @@
       "*": ["*", "src/*"],
       "~/*": ["./src/*"],
       // Add aliases from babel.config.js here if you want Intellisense for them
-      "@department-of-veterans-affairs/platform-forms-system/*": ["./src/platform/forms-system/src/js/*"],
-      "@department-of-veterans-affairs/platform-forms/*": ["./src/platform/forms/src/js/*"],
+      "@department-of-veterans-affairs/platform-forms-system/*": [
+        "./src/platform/forms-system/src/js/*"
+      ],
+      "@department-of-veterans-affairs/platform-forms/*": [
+        "./src/platform/forms/src/js/*"
+      ]
```

Comment:
When I merged in main and tried to push the changes, the trailing comma here was causing the pre-commit hook to fail with a linting error, so I removed it. 🤷‍♀️